9.4/paper.bib

```diff
@@ -636,10 +669,11 @@ @Misc{symengine-web-page
   title =        {{SymEngine: fast symbolic manipulation library, written in C++}},
   note =         {\url{https://symengine.org/}}}
 
+See: http://portal.hdfgroup.org/display/knowledge/How+do+I+properly+cite+HDF5+in+a+paper
 @Misc{hdf5,
     author = {{The HDF Group}},
     title = "{Hierarchical Data Format, version 5}",
-    year = {1997-2018},
+    year = {1997-2022},
```

Comment:
This would have to be `1997--2002`, but this isn't right. `year` needs to be a year, not a range. Just choose 2022.

Reply:
The HDF5 group specifically asks for the year range [here](http://portal.hdfgroup.org/display/knowledge/How+do+I+properly+cite+HDF5+in+a+paper).

I agree that it should not be a range. I will set it to 2022.